### PR TITLE
CNN

### PR DESCRIPTION
**Pull request by rehankarthikchandralal**
A branch was developed for task_4 - for Development of a Convolutional Neural Network.

The CNN was successfully trained and the training and validation loss curves were plotted

### CODE REVIEW

**Comment by rehankarthikchandralal** [comment]
Merging Pull request for CNN
